R/derive_param_computed.R

```diff
@@ -224,7 +264,20 @@
 #'   keep_nas = exprs(ADTF)
 #' )
 #'
-#' # Example 2: Derive BMI where height is measured only once
+#' @caption Example 2 - Using the `constant_parameters` argument
+#'
+#' @info Derive BMI where height is measured only once
+#'
+#' - Here, the parameters that are measured only once (i.e., the constant parameters)
```

Comment:
I think we need a bit more body here: Something like (without repeating the text in the roxygen code above too much):

"In the above examples, for each parameter specified in the `parameters` argument we expect one record per by group, where the by group is specifed in `by_vars`. However, if a parameter is only measured once, it can be specified in `constant_parameters` instead. We still need to supply a modified by group for the constant parameters, which can be done through `constant_by_vars`. See the example below, where weight is measured for each patient at each visit (`by_vars = exprs(USUBJID, VISIT)`) while height is measured for each patient only at the first visit (`constant_parameters = "HEIGHT"`, `constant_by_vars = exprs(USUBJID)`).